### ISSUE TITLE
"Current Annotation" colour is not themed

### ISSUE THREAD

**Issue by kerryland** [sumatrapdfreader/sumatrapdf#3675]
https://github.com/sumatrapdfreader/sumatrapdf/pull/3661 uses hard-coded "Black" border to indicate the "Current Annotation", so when a "Dark" theme is used it is practically impossible to see.

The "Current Annotation" border colour should be part of the theme definition.

**Comment by kjk** [comment]
makes sense. although maybe we can re-use text color, which should have contrast against background color, to avoid adding more stuff to theme.

I've looked at theme definitions of visual studio code and there's remarkably small number of distinct colors in the them.

I'm working on reducing how many things a theme defines.

In many cases I think we can re-use main text / background color with maybe accents using AdjustLightness2() etc.

**Comment by GitHubRulesOK** [comment]
thick black outline was highly confusing and youmb proposal uses  the conventional Cyan (as seen around highlights) but thick dots which is noticeable but not too heavy.

**Comment by kjk** [comment]
Using textColor for outline seems good enough in all themes

**Comment by meteorquake** [comment]
I see this is closed but I don't think it's sorted for me.
* Can I ask there be an option where it's either not shown or the same colour as the fill (yellow) is used for the border, either way rendering it invisible, or simply allow the user to choose the border colour manually.
Currently, the black border on OCR'd scans obscures the text you highlight yellow, because OCR text does not align with scan text, and this will also happen if you choose text colour. Even if it didn't obscure the text, I wouldn't want a border at any time since I functionally use them on the fly as a highlight and discard them, so a border is an eyesore!!

**Comment by GitHubRulesOK** [comment]
@meteorquake 
Totally agree the current form is NOT "eye candy" but the use of border is common here in Acrobat
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/5ad7a835-dc2c-4886-9818-b2cdfa868111) **PDFXedit**  ![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/391c8af5-ca0c-4ac2-8796-3b1054eaa62a) 
**Edge with Acrobat**   
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/91f4d831-34f4-4d88-9157-cce6ea8abe15)

**Comment by meteorquake** [comment]
Those are editors so aesthetics are less important since they represent design constructs.
By contrast Sumatra is a viewer, so it does need to be aesthetic, the highlights are going to be used for viewing rather than simply design.

**Comment by kerryland** [comment]
There will be no border at all once https://github.com/sumatrapdfreader/sumatrapdf/issues/3732 has been implemented, so I think the problem described by @meteorquake will go away.

Also, I'm using it to create and edit annotations, so *I'd* call it an editor :-)

**Comment by meteorquake** [comment]
Super, I'm happy at the concepts there!
Of course I agree sometimes it could be a very light editor so it's interesting you're perhaps using it sufficiently for that, I've always switched to Foxit Reader when I want to do any marking up but find that's unnecessarily heavyweight for viewing for which I use Sumatra...
d

**Comment by GitHubRulesOK** [comment]
not wanting to have a sytax fight but was using Foxit Reader for another question site reason and this is that "Readers" current highlight apply or edit mode 
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/da2e5498-bec2-47fc-b4dd-31dd07ffffa8)

**Comment by meteorquake** [comment]
Yes Foxit does that on creating the highlight but as soon as you click off it into the text that border disappears just giving an unbordered highlight. In Sumatra (currently!) when you click off into the text the border remains. :)

**Comment by meteorquake** [comment]
Hello! I've just installed an update and now when I press A it pops up a side box I have to click off. The black border does then disappear if I click that closed (it doesn't if I just click away) but is there any way of not having that box popup - or is it in the pipeline not to - as it's a killer of highlighting tasks where you just want to repeatedly highlight-and-colour...

**Comment by kjk** [comment]
@meteorquake that was a regression.

In next build `a`, `u` will not open the window, `A`, `U` will open the window.
